### DISCUSSION TITLE
Use username or other custom field to login

### DISCUSSION THREAD

**Discussion by clerisonb** [MarJose123/filament-lockscreen#29] [Q&A] [answered]
Is it possible to customize the field used to authenticate in the lock screen? I'm using a custom field, but the plugin seems to look for "email" only.

**Comment by MarJose123** [comment]
Are you using the default login page of Filament?

**Comment by clerisonb** [comment]
I'm using Breezy, configured to use a different field than email to
authenticate.

Em ter., 30 de mai. de 2023 22:34, MarJose ***@***.***>
escreveu:

**Comment by MarJose123** [comment]
> I'm using Breezy, configured to use a different field than email to
> authenticate.
> 
> Em ter., 30 de mai. de 2023 22:34, MarJose ***@***.***>
> escreveu:
> 
> > Are you using the default login page of Filament?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/MarJose123/filament-lockscreen/discussions/29#discussioncomment-6044232>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AHUELDJHZ74TSNUXVBTCPWTXI2NZZANCNFSM6AAAAAAYUVGQYU>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > <MarJose123/filament-lockscreen/repo-discussions/29/comments/6044232@
> > github.com>
> >
> 

I will check on what I can do. Feel free to PR, I'm happy to merge it.

**Comment by MarJose123** [comment, marked as answer]
You can now customize the login to a custom field by setting up the lockscreen config.
